R/zzz.R

```diff
@@ -55,6 +56,7 @@ default_linters <- modify_defaults(
 #'  * [debugonce()] is only useful for interactive debugging. It should be removed.
 #'  * [detach()] modifies the global search path. Detaching environments from the search path is rarely necessary in
 #'    production code.
+#'  * [testthat::expect_that()] This style of testing is no longer encouraged by `{testthat}` package. Using a more specific expect_ function (e.g. `expect_equal()`, `expect_s3_class()`, etc.) is recommended.
```

Comment:
line width

Reply:
Yeah, I wasn't sure about that because the other items in the list don't seem to follow 80-width rule.

At any rate, I've changed the new entry to follow this rule.

Reply:
we do 80 kinda organically, our max is 120 though